Doc/library/math.rst

```diff
@@ -355,6 +355,11 @@ Power and logarithmic functions
    of natural logarithms.  This is usually more accurate than ``math.e ** x``
    or ``pow(math.e, x)``.
 
+.. function:: exp2(x)
+
+   Return *2* raised to the power *x*.
+
+   .. versionadded:: 3.11
```

Comment:
One last nitpick-level suggestion: we're separating the various functions from each other by two blank lines in this file. I have no idea why, but let's keep the consistency.

I don't want to hold this PR up for such a trivial change, so I'll self-merge this change before merging the PR (provided the continuous integration is happy, of course).

```suggestion

.. function:: exp2(x)

   Return *2* raised to the power *x*.

   .. versionadded:: 3.11

```